assets/js/views/Config.vue

```diff
@@ -613,6 +630,9 @@ export default defineComponent({
 		vehicleOptions() {
 			return this.vehicles.map((v) => ({ key: v.name, name: v.config?.title || v.name }));
 		},
+		shmTags() {
+			return { configured: { value: true } };
```

Comment:
Hier wird im UI immer `Configured yes` angezeigt.
Ich kann mir vorstellen, dass das manche Benutzer verwirrt oder stört, weil `yes` angezeigt wird, obwohl man keine Konfiguration vorgenommen hat und sich der Wert nicht auf `false` zurücksetzen lässt.

Lässt sich die Rückgabe hier vielleicht an den Wert von `allowControl`

```suggestion
			return { allowControl: { value: store.state?.shm?.allowControl } };
```

oder aber an das Vorhandensein der beiden IDs (`deviceID`, `vendorID`) koppeln?

```suggestion
			const shm = store.state?.shm;
			return { configured: { value: !!shm?.deviceId && !!shm?.vendorId } };
```

Reply:
done